### PR TITLE
Update documentation to reflect dataset changes

### PR DESCRIPTION
**Pull request by shekelto**
Place IDs are now included in the dataset.

### CODE REVIEW

File: docs/table-search-trends.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ Regional CSV files are available for download in our [data exploration and downl
 | Name | Type | Description | Example |
 | ---- | ---- | ----------- | ------- |
 | **key** | `string` | Unique string identifying the region | US_CA |
+| **place_ID** | `string` | A textual identifier that uniquely identifies a place in the Google Places database and on Google Maps ([details](https://developers.google.com/places/web-service/place-id))| ChIJd_Y0eVIvkIARuQyDN0F1LBA |
```
Comment (owahltinez) [inline docs/table-search-trends.md:76]: Even though the Place ID is available at the source of data, users won't see it in our outputs for this table. Instead, PlaceID will be added to our index.
```suggestion
```

**Comment by shekelto** [comment]
Oscar, can you add this kind of text to your description of the region key, so people know what Place ID is?

**Comment by owahltinez** [comment]
Yes of course, we'll be adding a column named `place_id` to *all* of our data. We'll probably document it as part of the index table and mention it in the top-level README as well.

**Comment by owahltinez** [comment]
See #420 for a relevant change which adds PlaceID mapping to our locations

**Pull request merged**